src/main/java/ui/components/PanelMenuBar.java

```diff
@@ -186,6 +189,10 @@ private void augmentRenameableTextField(){
         menuBarNameArea.getChildren().addAll(renameableTextField);
         menuBarNameArea.setMinWidth(NAME_AREA_EDIT_WIDTH);
         menuBarNameArea.setMaxWidth(NAME_AREA_EDIT_WIDTH);
+        Tooltip.install(menuBarConfirmButton, new Tooltip("Confirm this name change"));
+        Tooltip undoTip = new Tooltip("Abandon this name change and go back to the previous name");
+        undoTip.setPrefWidth(220);
```

Comment:
This should probably be pulled out into a constant, since we may want similar wrapping dimensions for other tooltips.